test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_config_test.clj

```diff
@@ -136,19 +136,25 @@
              (merge valid-ssl-pem-config
                     {:ssl-host "foo.local"}))
            (munge-expected-https-config
-             {:https {:host "foo.local" :port default-https-port}})))
+             {:https {:host "foo.local"
+                      :port default-https-port 
+                      :allow-renegotiation default-allow-renegotiation}})))
```

Comment:
In general having to add this default to all of the https tests seems less than ideal. Does it have to be here for the tests to pass? If so can it and the other defaults be added to a defaults map that is merged with keys that we actually care about in each test?

Also, we should assert that you can set this to a non-default value, I guess. I'd expect to see using default and non-default values in the `munge-actual-https-config` section either an existing, or new, assertion.

Reply:
I have made some updates. pl let me know if I missed any test cases.